dir.props

```diff
@@ -100,6 +100,7 @@
     <DnuSourceList Include="https:%2F%2Fdotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <DnuSourceList Include="https:%2F%2Fdotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
     <DnuSourceList Include="https:%2F%2Fwww.myget.org/F/nugetbuild/api/v3/index.json" />
+    <DnuSourceList Include="https:%2F%2Fwww.myget.org/F/roslyn-nightly/api/v3/index.json" />
```

Comment:
right, actually I forgot about this, this is for the two packages that we needed to upload to our feed. I'll try to do it myself to see if I have permissions